101-data-factory-v2-onprem-sql-to-blob-copy/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dataFactoryName": {
+      "value": "GEN-UNIQUE"
+    },    
+    "dataFactoryLocation": {
+      "value": "East US"
+    },
+    "onPremSqlServerConnectionString": {
```

Comment:
remove this since it's not valid - use GEN-UNIQUE

Reply:
@bmoore-msft - you mean, the onPremSqlServerConnectionString? users need to enter it to deploy Data Factory entities successfully. Thanks.

Reply:
Fixed both. Thanks.